zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java

```diff
@@ -2041,9 +2063,10 @@ public NotebookInformationListener(NotebookServer notebookServer) {
     @Override
     public void onParagraphRemove(Paragraph p) {
       try {
-        notebookServer.broadcastUpdateNoteJobInfo(System.currentTimeMillis() - 5000);
+        getJobManagerService().getNoteJobInfoByUnixTime(System.currentTimeMillis() - 5000, null,
+            new JobManagerServiceCallback());
       } catch (IOException ioe) {
-        LOG.error("can not broadcast for job manager {}", ioe.getMessage());
+        LOG.warn("can not broadcast for job manager {}", ioe.getMessage());
```

Comment:
LOG.warn("can not broadcast for job manager {}", ioe.getMessage(), ioe);

Reply:
fixed